src/Admin/AbstractAdmin.php

```diff
@@ -397,6 +398,8 @@
 
     /**
      * @var array
+     *
+     * @deprecated Use TemplateRegistry service instead
```

Comment:
@greg0ire is this message correct?

Reply:
I think so, what bothers you with it?

Reply:
It is not. The deprecation should include "since 3.x, will be dropped in 4.0." first.